### ISSUE TITLE
Seems to not be creating table.

### ISSUE THREAD

**Issue by Gyztor** [Simonsator/Block-for-Party-and-Friends#1]
not sure what more to say other than i found this out when getting this error:
```
[23:56:57 ERROR]: java.sql.SQLSyntaxErrorException: (conn=54086) You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '-core.`fr_blocked` (`blocker_id` INT(8) NOT NULL, `blocked_id` INT(8) NOT NULL)' at line 1
[23:56:57 ERROR]:       at org.mariadb.jdbc.internal.util.exceptions.ExceptionFactory.createException(ExceptionFactory.java:62)
[23:56:57 ERROR]:       at org.mariadb.jdbc.internal.util.exceptions.ExceptionFactory.create(ExceptionFactory.java:153)
[23:56:57 ERROR]:       at org.mariadb.jdbc.MariaDbStatement.executeExceptionEpilogue(MariaDbStatement.java:273)
[23:56:57 ERROR]:       at org.mariadb.jdbc.ClientSidePreparedStatement.executeInternal(ClientSidePreparedStatement.java:229)
[23:56:57 ERROR]:       at org.mariadb.jdbc.ClientSidePreparedStatement.execute(ClientSidePreparedStatement.java:149)
[23:56:57 ERROR]:       at org.mariadb.jdbc.ClientSidePreparedStatement.executeUpdate(ClientSidePreparedStatement.java:181)
[23:56:57 ERROR]:       at de.simonsator.partyandfriends.block.BConnection.importTable(BConnection.java:29)
[23:56:57 ERROR]:       at de.simonsator.partyandfriends.block.BConnection.<init>(BConnection.java:20)
[23:56:57 ERROR]:       at de.simonsator.partyandfriends.block.BMain.onEnable(BMain.java:47)
[23:56:57 ERROR]:       at net.md_5.bungee.api.plugin.PluginManager.enablePlugins(PluginManager.java:356)
[23:56:57 ERROR]:       at net.md_5.bungee.BungeeCord.start(BungeeCord.java:320)
[23:56:57 ERROR]:       at net.md_5.bungee.BungeeCordLauncher.main(BungeeCordLauncher.java:54)
[23:56:57 ERROR]:       at net.md_5.bungee.Bootstrap.main(Bootstrap.java:15)
[23:56:57 ERROR]: Caused by: org.mariadb.jdbc.internal.util.exceptions.MariaDbSqlException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '-core.`fr_blocked` (`blocker_id` INT(8) NOT NULL, `blocked_id` INT(8) NOT NULL)' at line 1
[23:56:57 ERROR]:       at org.mariadb.jdbc.internal.util.exceptions.MariaDbSqlException.of(MariaDbSqlException.java:34)
[23:56:57 ERROR]:       at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.exceptionWithQuery(AbstractQueryProtocol.java:194)
[23:56:57 ERROR]:       at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.exceptionWithQuery(AbstractQueryProtocol.java:177)
[23:56:57 ERROR]:       at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.executeQuery(AbstractQueryProtocol.java:321)
[23:56:57 ERROR]:       at org.mariadb.jdbc.ClientSidePreparedStatement.executeInternal(ClientSidePreparedStatement.java:220)
[23:56:57 ERROR]:       ... 9 more
[23:56:57 ERROR]: Caused by: java.sql.SQLException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '-core.`fr_blocked` (`blocker_id` INT(8) NOT NULL, `blocked_id` INT(8) NOT NULL)' at line 1
[23:56:57 ERROR]:       at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.readErrorPacket(AbstractQueryProtocol.java:1674)
[23:56:57 ERROR]:       at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.readPacket(AbstractQueryProtocol.java:1536)
[23:56:57 ERROR]:       at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.getResult(AbstractQueryProtocol.java:1499)
[23:56:57 ERROR]:       at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.executeQuery(AbstractQueryProtocol.java:318)
[23:56:57 ERROR]:       ... 10 more
```
checked to make sure and the database `fr_blocked` is not being created

**Comment by Simonsator** [comment]
The plugin currently does not escape the database name correctly. Please use a database name without "-" for now.

I might not be able to fix this until Monday.

**Comment by Gyztor** [comment]
Alright, sounds good thank you for letting me know :)

**Comment by Simonsator** [comment]
Fixed with commit #a7c62e97a1c1bbfc55717f86f80abf2cb66b72b0
